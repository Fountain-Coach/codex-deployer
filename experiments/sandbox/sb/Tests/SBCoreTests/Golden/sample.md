# Sample

Paragraph text.

<!-- © 2025 Contexter alias Benedikt Eickhoff 🛡️ All rights reserved. -->
