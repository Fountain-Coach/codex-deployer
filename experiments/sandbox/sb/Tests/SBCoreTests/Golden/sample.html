<!doctype html>
<html><body><h1>Sample</h1><p>Hello</p><pre><code>print("hi")</code></pre><table><tr><th>A</th></tr><tr><td>1</td></tr></table></body></html>
<!-- © 2025 Contexter alias Benedikt Eickhoff 🛡️ All rights reserved. -->
